meta-jsdelivr/recipes-jsdelivr/jsdelivr-service/files/jsdelivr-docker-controller.py

```
@@ -0,0 +1,791 @@
+    """
+    Returns the raw output of docker logs for a specific container.
+
+    The command executed is:
+    docker logs $(docker ps --all --latest --filter 'name=<name>' --quiet) --since <since>
+
+    Query parameters:
+    - since: Optional timestamp or relative time (e.g., '2023-01-01T00:00:00', '1h', '30m')
+    """
+    try:
+        # First, get the container ID using docker ps with filter
+        ps_result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)